daprdocs/content/en/operations/configuration/configuration-overview.md

```diff
@@ -86,9 +86,9 @@ The `tracing` section under the `Configuration` spec contains the following prop
 tracing:
   samplingRate: "1"
   otel: 
-    endpointAddress: "https://..."
+    endpointAddress: "otelcollector.observability.svc.cluster.local:4317"
   zipkin:
-    endpointAddress: "http://zipkin.default.svc.cluster.local:9411/api/v2/spans"
+    endpointAddress: "zipkin.default.svc.cluster.local:9411/api/v2/spans"
```

Comment:
Just a heads up that `zipkin` does need the protocol in the endpoint address. It's only when using `otel` that it doesn't.

Reply:
Updated